libkbfs/key_manager.go

```diff
@@ -733,24 +757,32 @@ func (km *KeyManagerStandard) Rekey(ctx context.Context, md *RootMetadata, promp
 		}
 	}()
 
+	// From this point on, if we return true for mdChanged with a
+	// nil error or RekeyIncompleteError{}, we must call
+	// km.finalizeRekey(md) first.
+
+	defer func() {
+		if mdChanged && (err == nil || err == RekeyIncompleteError{}) {
```

Comment:
this `RekeyIncompleteError` check could be brittle if we ever add fields to it; maybe check the type instead?